pandas/io/tests/test_pytables.py

```diff
@@ -2665,23 +2665,6 @@ def test_sparse_frame(self):
         self._check_double_roundtrip(ss3, tm.assert_frame_equal,
                                      check_frame_type=True)
 
-    def test_sparse_panel(self):
```

Comment:
no a search is not sufficient
but if we don't have tests which fail when u remove it - it is fine

you always have to assume other people don't do things and test everything